snake/stage.py

```diff
@@ -13,9 +13,10 @@
 import __main__
 import os
 import time
+import gdk
 
 from kano.window import _get_window_by_child_pid, gdk_window_settings
-from kano_settings.system.display import get_status
+from kano.logging import logger
```

Comment:
Kano.logging takes a second to import, you might want to import it where you generate an error message.